superset-frontend/src/SqlLab/components/SqlEditor/index.tsx

```diff
@@ -284,6 +284,9 @@ const SqlEditor: React.FC<Props> = ({
   const [autocompleteEnabled, setAutocompleteEnabled] = useState(
     getItem(LocalStorageKeys.SqllabIsAutocompleteEnabled, true),
   );
+  const [renderHTMLEnabled, setRenderHTMLEnabled] = useState(
+    getItem(LocalStorageKeys.SqllabIsRenderHtmlEnabled, true),
```

Comment:
same here. Please set `false` for fallback case.
```suggestion
    getItem(LocalStorageKeys.SqllabIsRenderHtmlEnabled, false),
```